### PR TITLE
Deprecate Helm Hub chart

### PR DESCRIPTION
**Pull request by chrisjs**
Resolves #4063

### CODE REVIEW

**Comment by chrisjs** [comment]
@ilayaperumalg i can take care of modifying the CI plans once this is merged.

File: README.adoc [hunk 1]
```diff
@@ -30,6 +30,8 @@ export KUBECONFIG=config location
 export CLUSTER_NAME=<cluster-name>
 #By default the setup will perform a helm install. Uncomment to install directly from the git repo.
 #export USE_DISTRO_FILES=true
+#From SCDF 1.6 and above, the bitnami chart should be used. For below, the legacy Helm stable repo can be used:
```
Comment (ilayaperumalg) [inline README.adoc:33]: I think you meant the version 2.6 ;) ?

File: kubernetes/skipper-server/create.sh [hunk 2]
```diff
@@ -28,10 +29,20 @@ function use_helm() {
...
+  if [ -z "$USE_LEGACY_HELM_CHART" ]; then
+    helm repo add bitnami https://charts.bitnami.com/bitnami
+  fi
+
   helm repo update
 
-  helm install --name scdf stable/spring-cloud-data-flow ${HELM_PARAMS} --namespace $KUBERNETES_NAMESPACE
+  if [ ! -z "$USE_LEGACY_HELM_CHART" ]; then
+    HELM_CHART_REFERENCE="stable/spring-cloud-data-flow"
+	echo "WARN: Using legacy helm chart, bitnami chart should be used as of SCDF 1.6"
```
Comment (ilayaperumalg) [inline kubernetes/skipper-server/create.sh:42]: version 2.6

Comment (ilayaperumalg) [reply kubernetes/skipper-server/create.sh:42]: will fix when merging.

**Comment by ilayaperumalg** [comment]
LGTM, fixed the typo and merged as 330cfb8

**Pull request closed without merging**